### PR TITLE
Add sphinx-sitemap to docs.

### PR DESCRIPTION
**Pull request by jezdez**
Refs https://github.com/conda/conda-docs/issues/810.

### Checklist - did you ...

- [ ] ~Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?~
     
- [ ] ~Add / update necessary tests?~
- [x] Add / update outdated documentation?

### CODE REVIEW

**Comment by jezdez** [comment]
Example link: https://continuumio-conda-docs--11697.com.readthedocs.build/projects/conda/en/11697/sitemap.xml

File: docs/source/conf.py [hunk 1]
```diff
@@ -162,6 +164,9 @@
     ]
 }
 
+html_baseurl = 'https://docs.conda.io/projects/conda/'
+sitemap_locales = [None]
+sitemap_url_scheme = "{lang}stable/{link}"
```
Comment (jezdez) [inline docs/source/conf.py:169]: We're hard-coding stable here since that's what we want Google to point to.

Comment (kenodegard) [reply docs/source/conf.py:169]: should probably add this as an inline comment?

Comment (jezdez) [reply docs/source/conf.py:169]: True!

**Review by kenodegard — APPROVED** [review]

**Review by kenodegard — APPROVED** [review]

**Pull request merged**